src/main/java/org/kohsuke/github/GitHub.java

```diff
@@ -205,6 +205,15 @@ public static GitHub connectAnonymously() throws IOException {
     }
 
     /**
+     * Connects to GitHub Enterprise anonymously.
```

Comment:
This methods existed before normal builder appeared, use builder from your code as valid way of connection creation.